contracts/SupportedAssets.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+
+/**
+ * @title SupportedAssets
+ * @dev Contract storing list of available assets for investing
+ */
+contract SupportedAssets is Ownable {
```

Comment:
I'm not sure if we should reinvent the wheel. Moreover storing the map as contract could be more expensive. How about using:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/structs/EnumerableMap.sol

Reply:
Let's move it to IAssetExchange and implement in PangolinExchange

Reply:
replaced with EnumerableMap from OpenZeppelin and moved